app/controllers/staff/training_sessions_controller.rb

```diff
@@ -0,0 +1,130 @@
+class Staff::TrainingSessionsController < StaffAreaController
+  before_action :load_user, only: [:show, :edit, :update]
+
+  layout 'staff_area'
+
+  def index
```

Comment:
I find this method really really confusing, maybe we can thin it out. Do we need all the pagination and sorting stuff? Maybe that can be added in another PR after this one is merged?